trackers_integration/tests/test_trac.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2022-2024 Alexander Todorov <atodorov@otb.bg>
+# Copyright (c) 2025 Frank Sommer <Frank.Sommer@sherpa-software.de>
+#
+# Licensed under GNU Affero General Public License v3 or later (AGPLv3+)
+# https://www.gnu.org/licenses/agpl-3.0.html
+
+# pylint: disable=attribute-defined-outside-init, protected-access
+import os
+
+from django.utils import timezone
+
+from tcms.core.contrib.linkreference.models import LinkReference
+from tcms.rpc.tests.utils import APITestCase
+from tcms.testcases.models import BugSystem
+from tcms.tests.factories import ComponentFactory, TestExecutionFactory
+
+from trackers_integration.issuetracker import trac
+from trackers_integration.issuetracker.trac import Trac
+
+
+class TestTracIntegration(APITestCase):
+    def __init__(self):
+        super().__init__()
+        self.project_name = os.getenv("TRAC_PRODUCT", "MyProject")
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
```

Comment:
Failing b/c in latest versions Django has changed this method. See https://github.com/kiwitcms/trackers-integration/pull/91 for the relevant changes necessary.